src/dashboard/pyproject.toml

```
@@ -0,0 +1,82 @@
+#
+# QualiCharge package
+#
+[project]
+name = "qualicharge-dashboard"
+version = "0.0.1"
+
+# Third party packages configuration
+[tool.coverage.run]
+omit = [
+  "migrations"
+]
+
+[tool.pytest.ini_options]
+addopts = "-v --cov-report term-missing --cov=apps"
+DJANGO_SETTINGS_MODULE = "apps.dashboard.settings"
+python_files = [
+    "test_*.py",
+    "tests.py",
+]
+testpaths = [
+    "tests",
+]
+
+[tool.ruff]
+line-length = 120
+indent-width = 4
+target-version = "py312"
```

Ditto? And eventually change all target version across services.

```suggestion
target-version = "py310"
```

---

✅